src/exampleTypes/ResultCount.js

```diff
@@ -4,15 +4,32 @@ import { observer } from 'mobx-react'
 import { withNode, withInlineLoader } from '../utils/hoc'
 import { toNumber } from '../utils/format'
 
-let ResultCount = ({ node, display = toNumber }) => {
-  let count = F.cascade(
-    ['context.response.results.length', 'context.results.length'],
-    node
-  )
+let ResultCount = ({
+  node = {},
+  display = toNumber,
+  noResults = 'No Results',
+}) => {
+  let count =
+    F.cascade(
+      [
+        'context.response.results.length',
+        'context.results.length',
+        'context.value',
+      ],
+      node
+    ) || _.isNumber(node.context)
```

Comment:
`node.context` should not be a scalar - what is this case for?

Reply:
I made the new `count` node's context a scalar.

Reply:
`count` is a scalar node. I asked about that before making this.

Reply:
@stellarhoof we need to unwind that. The next major version will likely drop `context` as a wrapper entirely.

Reply:
Changed the returned value to `{ result: # }`